### DISCUSSION TITLE
next.js 버전 MVP 사용자 배포!

### DISCUSSION THREAD

**Discussion by cyberjam** [LifeIsAlone/public-home#52] [General]
<img width="577" alt="image" src="https://user-images.githubusercontent.com/20448972/192696565-ee777bbe-e463-4a9c-b158-6c32f41db0bb.png">

**Comment by cyberjam** [comment]
아 ㅜ 영상에는 올라가지 못해 아쉽지만 다음 sh공고를 한번 노려보죠 ㅎㅎ

![Screenshot_20220928-153453_Instagram.jpg](https://user-images.githubusercontent.com/20448972/192705491-f1602d60-930c-4794-aaa9-0edc5de4a845.jpg)

**Reply by sangjuun-park** [reply]
구현하시느라 고생하셨습니다

**Comment by cyberjam** [comment]
![Screenshot_20221001-022950_NAVER.jpg](https://user-images.githubusercontent.com/20448972/193324918-4086b117-0dee-433c-bda5-77aa0cb87891.jpg)

그래도 우리 서비스 잊지 않고 찾아주시는 분이 계셔서 보람있군요
하루빨리 seo최적화해서 네이버 첫번째에 똭 올려놔야겠어요
